test/functional/plugin/lsp/completion_spec.lua

```diff
@@ -218,6 +218,49 @@ describe('vim.lsp.completion: item conversion', function()
     eq(expected, result)
   end)
 
+  it('removes unnecessary symbols in label', function()
+    local items = {
+      {
+        detail = 'colnr_T',
+        filterText = 'w_virtcol',
+        insertText = '->w_virtcol',
+        insertTextFormat = 1,
+        kind = 5,
+        label = ' w_virtcol',
```

Comment:
why is this preceded by whitespace (also the `abbr` in the result below)? The test just above this one tests that whitespace is supposed to be trimmed...

Reply:
This is the result of lsp clangd returns some results with `•function_name` The space is to distinguish the period. space in there is necessary.